tools/capture-test.py

```diff
@@ -295,7 +298,9 @@ def cap_to_playback(buf):
     # treat it, and it can plausibly create false positive chirp signals
     # loud enough).
     for i in range(0, len(buf), capsz):
-        frame = [scale * x for x in struct.unpack(capfmt, buf[i:i+capsz])[0:opts.chan]]
+        frame = struct.unpack(capfmt, buf[i:i+capsz]) # Decode
+        frame = [frame[i] for i in opts.capmap]       # Select via channel map
```

Comment:
Please don't re-use `i` again inside `for i`, that really threw me off! How about `c` or even better: `chan`?